### PR TITLE
feat(loqusDB): Adds new loqusDB exports for snv/indels and SVs

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- See above

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
@jemten Think I am done and this is ready for review

**Comment by jemten** [comment]
Looks to be some issues still :/

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 **References**
 - grch38_frequency_vcfanno_filter_config_-v1.2-.toml -> grch38_frequency_vcfanno_filter_config_-v1.3-.toml
 - grch37_frequency_vcfanno_filter_config_-v1.3-.toml -> grch37_frequency_vcfanno_filter_config_-v1.4-.toml
+- grch37_frequency_vcfanno_filter_config_-v1.3-.toml -> grch37_frequency_vcfanno_filter_config_-v1.5-.toml
```
Comment (jemten) [inline CHANGELOG.md:28]:
```suggestion
- grch37_frequency_vcfanno_filter_config_-v1.4-.toml -> grch37_frequency_vcfanno_filter_config_-v1.5-.toml
```

Comment (henrikstranneheim) [reply CHANGELOG.md:28]: You are completely right, but next PR. I also have to change the download part and I would like to do it sequentially to be able to track it better

**Pull request merged**